examples/moondream-chatbot/bot.py

```diff
@@ -68,7 +68,7 @@
 
 class TalkingAnimation(FrameProcessor):
     """This class starts a talking animation when it receives an first AudioFrame,
-    and then returns to a "quiet" sprite when it sees a TTSStoppedFrame.
+    and then returns to a "quiet" sprite when it sees a BotStartedSpeakingFrame.
```

Comment:
Shouldn't this be stopped speaking?

```suggestion
    and then returns to a "quiet" sprite when it sees a BotStoppedSpeakingFrame.
```